### ISSUE TITLE
viteBundler doesn't work on blank install

### ISSUE THREAD

**Issue by ericuldall** [payloadcms/payload#4247]
### Link to reproduction

https://github.com/ericuldall/payload-vite-broken

### Describe the Bug

When running it starts fine in the console but when I browse to localhost:3000 I get the following error in console:
```
payload.config.ts:25 Uncaught TypeError: Cannot read properties of undefined (reading 'DATABASE_URI')
    at src/payload.config.ts (payload.config.ts:25:22)
    at __init (chunk-FOWDQQBS.js?v=8dcf3535:16:56)
    at node_modules/payload/dist/admin/Root.js (Root.tsx:75:1)
    at __require2 (chunk-FOWDQQBS.js?v=8dcf3535:19:50)
    at node_modules/payload/components/root.js (root.ts:1:21)
    at __require2 (chunk-FOWDQQBS.js?v=8dcf3535:19:50)
    at root.ts:1:33
```

### To Reproduce

Using node v18 on macos:

- Run `npm i`
- Run `npm run dev`
- Browse to http://localhost:3000

### Payload Version

2.0.0

### Adapters and Plugins

blank

**Comment by haliacmon** [comment]
It seams vite can't load any enviroment variables. DATABASE_URI is just happens to be the first env import in payload.config.ts

**Comment by PatrikKozak** [comment]
Hey @ericuldall and @haliacmon - we just released a new version of the `@payloadcms/bundler-vite` package with a fix for this issue - `0.1.5`.

Just tested it out on your reproduction repo and no issues anymore - just update the package and you should be all set. Let me know how it goes!
